<?xml version="1.0"?>
<launch>

    <!-- Parameters for path follower -->
    <arg name="data_name" default="test.txt"/>
    <arg name="data_mode" default="joints"/>
    <arg name="arm_prefix" default="right_"/>

    <node name="follow_path" pkg="path_follower" type="follow_path.py" output="screen">
        <param name="data_name" value="$(arg data_name)"/>
        <param name="data_mode" value="$(arg data_mode)"/>
        <param name="arm_prefix" value="$(arg arm_prefix)"/>
    </node>

</launch>
